src/regmod/variable.py

```diff
@@ -18,15 +18,75 @@
 
 @dataclass
 class Variable:
+    """Variable class is in charge of storing information of variable including
+    name and priors, and accessing data in the data frame. Name correspondes to
+    column name in the data frame and priors are used to compute the likelihood.
+
+    Parameters
+    ----------
+    name : str
+        Name of the variable corresponding to the column name in the data frame.
+    priors : List[Prior], optional
```

Comment:
I'm wondering how the priors are used to compute the likelihood. Is it priors for the betas?

Reply:
Yes! the priors are directly on the variables in the likelihood.